LSE-61.tex

```diff
@@ -1174,6 +1178,93 @@ \subsubsection{All-Sky Visualization of Data Releases}
+\textbf{ID:} DMS-REQ-0379
+
+\textbf{Specification:}
+Data Release Production shall include the production of an all-sky image map for the existing coadded image area in each filter band, and at least one pre-defined all-sky color image map, following the IVOA HiPS Recommendation.
+
+\textbf{Discussion:}
+The maximum resolution of the image maps is TBD; however, it would be desirable for it to be at least close to the underlying coadded image resolution, in order not to give a poor impression of the data quality.  It is possible that the highest-resolution HiPS tiles could be provided on-demand from the LSST cutout service.  It is expected that the HiPS tiles will be generated by resampling the existing coadds, not by performing an independent coaddition.  This requires work from Science Pipelines on resolving the ambiguities in overlap regions.  Whether the lower-resolution levels of the HiPS tiles will be generated by existing community tools (i.e., hipsgen) or by LSST code is also TBD.  The color map being "pre-defined" means that the choice of bands will be made by the LSST Project as part of the configuration of a Data Release.  This does not preclude the Science Platform additionally providing means for interactive generation of other colorizations from the single-band HiPS maps.
+
+<br>By the terms of the HiPS Recommendation, a HiPS image map should include a corresponding MOC.  This may or may not be the same as the MOCs for the survey envisioned under \hyperref[DMS-REQ-0383]{DMS-REQ-0383} elsewhere in this document, depending on choices made for data selection.
+
+<br>The Project should produce a technical note, during the construction era, detailing which of the optional components of the HiPS standard will be supported.
+
+<br>This requirement specifically calls for making HiPS maps from the standard coadds and therefore whatever policies are used for the inclusion of Special Programs data in the standard coadds will also automatically apply here.  If there are both main-survey-depth and full-depth coadds for the deep drilling fields, then, it is a separate question as to whether HiPS maps will be generated for those fields.
+
+\paragraph{HiPS Service}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0380}
+\textbf{ID:} DMS-REQ-0380
+
+\textbf{Specification:}
+The Data Management system shall include a secure and authenticated Internet endpoint for an IVOA-compliant HiPS service.  This service shall be advertised via Registry as well as in the HiPS community mechanism operated by CDS, or whatever equivalent mechanism may exist in the LSST operations era.
+
+\textbf{Discussion:}
+The DM HiPS service will be available only to data rights holders.  LSST EPO will also operate a world-public HiPS service, but with its spatial resolution limited to approximately one arcsecond.
+
+\paragraph{HiPS Linkage to Coadds}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0381}
+\textbf{ID:} DMS-REQ-0381
```

Comment:
Priority 2 and OSS-REQ-0061 and OSS-REQ-0122 (Provenance).  (Priority 2 because, given a set of coordinates on the HiPS map, a query to the coadded image database can be made independently.  However, this feature remains highly desirable.)